rp2040-hal/examples/spi.rs

```diff
@@ -89,7 +89,7 @@ fn main() -> ! {
         &mut pac.RESETS,
         clocks.peripheral_clock.freq(),
         16.MHz(),
-        embedded_hal::spi::MODE_0,
+        embedded_hal_0_2::spi::MODE_0,
```

Comment:
This should use HAL 1.0

Reply:
:+1: 
I relied upon the compiler notifying me if the 0.2 types were still used. But as `new` only requires `Into<FrameFormat>`, and that is implemented for both 0.2 and 1.0 types, no error was shown.